draft-ietf-quic-multipath.md

```diff
@@ -1333,14 +1304,15 @@ MP_RETIRE_CONNECTION_ID Frame {
 ~~~
 {: #fig-mp-retire-connection-id-frame-format title="MP_RETIRE_CONNECTION_ID Frame Format"}
 
-Path Identifier:
-: A Path ID which is pre-allocated when the connection ID is generated, which
-  means the current connection ID can only be used on the corresponding path.
+Compared to the RETIRE_CONNECTION_ID frame specified in
+{{Section 19.16 of QUIC-TRANSPORT}}, the following
+field is added:
 
-Sequence Number:
-: The sequence number assigned to the connection ID by the sender for the path
-  specified by Path ID, encoded as a variable-length integer.
+Path Identifier:
+: The Path ID associated with the connection ID to retire.
 
```

Comment:
Note that the Sequence Number also has different meaning here. It's defined for per path.

Reply:
That do matters. When people are implementing the MP_CID_RETIRE frame, they could get all the important changes within this section, don't necessarily need to jump to the MP_NEW_CID frame definition section.

Reply:
But it's not a change. you look up the Path ID then you find the sequence number and remove the CID. So the only change is that you have to look up the Path ID first, which is described. That the sequence number is path-specific is only important when you select one in the NEW_CID frame.

Reply:
I agree with Mirja here. However, we could probably add a sentence in the above paragraph making the link with the "sequence number on a per-path context", similarly to what was done with MP_NEW_CONNECTION_ID frame, to address Yanmei's concern.

Reply:
You can make a proposal but a sentence like that make less sense here as the sequence number is selected when the NEW_CID frame is set and here only used. So saying the sequence number is "applied" on a per-path basis doesn't really make sense. Maybe the sequence number has be applied on a per path basis when the NEW_CID was sent but that information is already in the other section and not really helpful here.